src/sentry/lang/native/plugin.py

```diff
@@ -152,6 +181,111 @@ def preprocess_step(self, processing_task):
         if options.get('symbolserver.enabled'):
             self.fetch_system_symbols(processing_task)
 
+        if self.use_symbolicator:
+            self.run_symbolicator(processing_task)
+
+    def run_symbolicator(self, processing_task):
+        # TODO(markus): Make this work with minidumps. An unprocessed minidump
+        # event will not contain unsymbolicated frames, because the minidump
+        # upload already happened in store.
+        # It will also presumably not contain images, so `self.available` will
+        # already be `False`.
+
+        if not self.available:
+            return
+
+        request_id_cache_key = request_id_cache_key_for_event(self.data)
+
+        stacktraces = []
+        processable_stacktraces = []
+        for stacktrace_info, pf_list in processing_task.iter_processable_stacktraces():
```

Comment:
Doesn't this also now contain javascript frames for RN?

Reply:
It checks `if pf.processor == self` below, so it should be filtered correctly.